Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml

```diff
@@ -0,0 +1,12 @@
+- type: construction
+  name: Brass knuckles
+  id: BrassKnuckles
+  graph: GraphWeaponBrassKnuckles
+  startNode: start
+  targetNode: WeaponBrassKnuckles
+  category: construction-category-weapons
+  description: Sometimes there is only one way to shut a mouth that overstayed their welcome
```

Comment:
1. Why use a different description here?
2. Strangely phrased.  A mouth can't move on its own.

Why even have a construction graph for these?  They aren't some improvised weapons.

Reply:
because checkraze requested it.  also, yes everyone knows a mouth cant move on it own. it means someone is talking to much shit

Reply:
the description could be better, I agree, but when discussing it on discord I thought it would fit in with the hand-made weaponry like the baseball bat, folding chair, or spear